src/client/app/graph/canvas/canvas.service.ts

```diff
@@ -14,6 +14,11 @@ export type ViewportCoord = Point;
 // A point in the coordinate system of the canvas.
 export type CanvasCoord = Point;
 
+// Specifies the type of connection an arrow has to a card.
```

Comment:
If we do need this enum, we should clarify that this enum is _relative_. See if you can clarify that these enum values could actually both be "applied" to the same arrow simultaneously (an arrow can point from one card and into another).

Reply:
Done